backend/src/main/kotlin/de/neuefische/backend/user/UserService.kt

```diff
@@ -0,0 +1,20 @@
+package de.neuefische.backend.user
+
+import org.springframework.stereotype.Service
+
+@Service
+class UserService(private val userRepo: UserRepo) {
+
+    fun getUserByName(name: String): UserProfileResponse? {
+        return userRepo.findByName(name)?.let { user ->
+            UserProfileResponse(
+                id = user.id.value.toString(),
```

Comment:
Does this work if id was an object? I'm no expert though... so ignore this suggestion if it's wrong! 😅
```suggestion
                id = user.id.toHexString(), 
```

Reply:
value.toString works :-) still haven't tried the other way round though (get by id)

```
curl -X GET "http://localhost:8080/api/user/john_doe" -H "Content-Type: application/json"
{"id":"652d1f1e8f1b2c3d4e5f6a7b","name":"john_doe","bio":"Software developer with a passion for open-source projects.","imageUrl":"https://res.cloudinary.com/the-work-of-art/image/upload/cld-sample-4.jpg","mediums":["watercolor"]}% 
```

Reply:
the toHexString didn't work for me actually (I also tried it)